packages/website/src/components/ast/PropertyValue.tsx

```diff
@@ -1,34 +1,92 @@
-import React from 'react';
+import Link from '@docusaurus/Link';
+import React, { useMemo, useState } from 'react';
 
 import styles from './ASTViewer.module.css';
-import type { ASTViewerModelMap } from './types';
+import { objType } from './utils';
 
 export interface PropertyValueProps {
-  readonly value: ASTViewerModelMap;
+  readonly value: unknown;
+}
+
+export interface SimpleModel {
+  readonly value: string;
+  readonly className: string;
+  shortValue?: string;
```

Comment:
[Question] Is this one not able to be `readonly`?

Reply:
correct, 

its muted in

```
  const model = useMemo(() => {
    const val = getSimpleModel(value);
    if (val.value.length > 250) {
      val.shortValue = val.value.substring(0, 200);
    }
    return val;
  }, [value]);
```

Reply:
i applied this suggestion :)